packages/camera/camera/ios/Classes/DeviceOrientation.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I like the idea of splitting out things from `CameraPlugin` but this seems excessive, each header only has like 2 functions/typedef in them, and the function implementations are super simple.  There's more boilerplate than logic in some of them.  Same with the tests, I don't think we need 5 new test files each containing ~10 lines of asserts.

There's no fast rules here, it's preference, but lots of little interrelated files are hard to navigate, too.

What about a `CameraProperties.h/m` with everything you split out?

Reply:
I like the `CameraProperties.h/m` approach. Will update.